src/main/java/com/salaboy/conferences/site/ProposalService.java

```diff
@@ -0,0 +1,31 @@
+package com.salaboy.conferences.site;
+
+import com.salaboy.conferences.site.model.Proposal;
+import com.salaboy.conferences.site.model.ProposalStatus;
+import com.salaboy.conferences.site.repository.ProposalRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Service
+public class ProposalService {
+
+    @Autowired
+    private ProposalRepository repository;
+
+    public Set<Proposal> getProposals(boolean pending) {
+        if (!pending) {
+
+            return new HashSet<>(repository.findAll());
```

Comment:
@mcruzdev I would prefer to add a filter going to the database instead of doing it in memory.. can you add that to the ProposalRepository?, it should be just adding an extra method to filter by the proposal status.